packages/common/src/projects/HubProjects.ts

```diff
@@ -186,7 +201,7 @@ export async function updateProject(
  * @param identifier item id or slug
  * @param requestOptions
  */
-export function getProject(
+export function fetchProject(
```

Comment:
is this a breaking change? do we need an alias for backwards compatibility?

Reply:
technically... yes. but nothing's using it yet... I'll see if I can add an alias w/o messing with interfaces